jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -1051,6 +1060,39 @@ public static void setProperties(Properties p) {
 	}
 }
 
+static void initSecurityManager(ClassLoader applicationClassLoader) {
+	String javaSecurityManager = internalGetProperties().getProperty("java.security.manager"); //$NON-NLS-1$
+	/*[IF JAVA_SPEC_VERSION > 11]*/
+	if ("allow".equals(javaSecurityManager)) {
```

Comment:
does RI do a case sensitive check?

Reply:
yes, RI follows the [Javadoc](https://cr.openjdk.java.net/~iris/se/18/spec/latest/api/java.base/java/lang/SecurityManager.html#java.security.manager) and performs a case sensitive check.